file-formats/uiad/type/zif_aff_uiad_v1.intf.abap

```diff
@@ -267,72 +267,77 @@ INTERFACE zif_aff_uiad_v1
 
     ty_signature_items TYPE STANDARD TABLE OF ty_signature_item WITH DEFAULT KEY.
 
-  "! <p class="shorttext">Additional Param Handling</p>
+  "! <p class="shorttext">Additional Parameter Handling</p>
   "! How additional parameters are handled
   "! $values {@link zif_aff_uiad_v1.data:co_additional_param_handling}
   "! $default {@link zif_aff_uiad_v1.data:co_additional_param_handling.allowed}
   TYPES ty_additional_param_handling TYPE c LENGTH 1.
 
   CONSTANTS:
-    "! <p class="shorttext">Additional Param Handling</p>
+    "! <p class="shorttext">Additional Parameter Handling</p>
     "! How additional parameters are handled
     BEGIN OF co_additional_param_handling,
 
       "! <p class="shorttext">Allowed</p>
-      "! Additional parameters are allowed.
+      "! Additional parameters are allowed
       allowed     TYPE ty_additional_param_handling VALUE 'A',
 
       "! <p class="shorttext">Ignored</p>
-      "! Additional parameters are ignored.
+      "! Additional parameters are ignored
       ignored     TYPE ty_additional_param_handling VALUE 'I',
 
       "! <p class="shorttext">Not Allowed</p>
-      "! Additional parameters are not allowed.
+      "! Additional parameters are not allowed
       not_allowed TYPE ty_additional_param_handling VALUE 'N',
 
     END OF co_additional_param_handling.
 
   TYPES:
-    "! <p class="shorttext">Navigation Details</p>
-    "! Navigation details
+    "! <p class="shorttext">Navigation Information</p>
+    "! Navigation information
     BEGIN OF ty_navigation,
 
-      "! <p class="shorttext">Target Mapping ID</p>
-      "! Target mapping ID
+      "! <p class="shorttext">Target Mapping</p>
+      "! Target mapping
+      "! $required
       target_mapping_id             TYPE c LENGTH 50,
 
-      "! <p class="shorttext">Target Url</p>
-      "! Target url used for 'Tile only' and 'Url app' apps
+      "! <p class="shorttext">Semantic Object</p>
+      "! Semantic object of the target mapping
+      "! $required
+      semantic_object               TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Action</p>
+      "! Action of the target mapping
+      "! $required
+      action                        TYPE c LENGTH 60,
+
+      "! <p class="shorttext">Target URL</p>
+      "! Target URL used for apps of type 'Tile Only' and 'URL Application'
       target_url                    TYPE string,
 
+      "! <p class="shorttext">System Alias</p>
+      "! System alias
+      system_alias      TYPE c LENGTH 32,
+
       "! <p class="shorttext">Desktop</p>
       "! Desktop is supported as device type
-      "! $required
+      "! $default 'X'
```

Comment:
@SAP/abap-file-formats-team  Is it correct to specify the value with `'X'`? Shouln't it be `true` or maybe `'true'` for boolean values.

It is not specified clearly here: https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#default-values

Reply:
I have just talked to the team. We usually do not recommend to use defaults for boolean values and therefore suggest to remove the default at all here. 

Reason: Usually we do not serialize (unless the field is required or showAlways) boolean values if they are false. During deserialization they are set to `false` in this case.
With a default as suggested above, the behavior would just been vice-versa.

Reply:
Regarding the default of the boolean field:
We have 3 checkboxes for for the supported device types: Desktop, Tablet, and Phone. All three are booleans and at least one must be selected. Therefore we thought that default 'X' of desktop is a good idea.
Alternatively we can implement a backend check that at least one of them is selected.

Reply:
Mmh, default in our case usually means. If you don't specify the field in the JSON it would be automatically set to the default value.

For other boolean values, if the value/field is not specified, it will be set to SPACE in ABAP which means `false`. I guess I wouldn't add a special boolean in your case with the opposite behavior.

A semantic check (to verify at least one option is set) is fine. I would add this if you can. I think there would be even options to specify this with JSON schema. Unfortunately, we haven't supported these constructs, yet. :(

Reply:
We will remove default from interface and set default values is done with backend. If you create a new UIAD object checkbox for all device type are set.
I will commit a corrected files soon.